### DISCUSSION TITLE
Disable the selection of a specific day of the week

### DISCUSSION THREAD

**Discussion by appcarrareto** [SashaJarvi/vue-inline-calendar#5] [General]
I would like to know how to disable the selection of a specific day of the week, for example by sending an array of the corresponding weekdays to be disabled, like [0,1,3].

0 = Sunday
1 = Monday
3 = Wednesday

**Comment by SashaJarvi** [comment]
Hello! Thanks for your question. I've re-read my docs and code, and find out that you can disable weekends only and past/next days, and there is no the opportunity to disable specific days. I think I should add this opportunity – it wiil be soon. Thank you once again.
